CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFileManager+FileManagement.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  CEWorkspaceFileManager+FileSystem.swift
+//  CodeEdit
+//
+//  Created by Khan Winter on 9/30/23.
+//
+
+import Foundation
+import AppKit
+
+extension CEWorkspaceFileManager {
+    /// This function allows creation of folders in the main directory or sub-folders
+    /// - Parameters:
+    ///   - folderName: The name of the new folder
+    ///   - file: The file to add the new folder to.
+    /// - Authors: Mattijs Eikelenboom, KaiTheRedNinja. *Moved from 7c27b1e*
```

Comment:
Don't know if these "authors" things are needed, because if changes are made by other people this won't be correct anymore. (Don't know if you added them or if they where already present btw)